src/main/java/sh/damon/stackmob/command/commands/CreateStackEntity.java

```diff
@@ -65,20 +65,19 @@ public int run(CommandContext<ServerCommandSource> context) throws CommandSyntax
         if (entity == null)
             throw new SimpleCommandExceptionType(new LiteralMessage("Failed to create entity")).create();
 
-        if (!EntityHelper.spawnEntity(entity))
-            throw new SimpleCommandExceptionType(new LiteralMessage("Failed to create entity, UUID duplicate in registry.")).create();
-
         final StackMob sm = StackMob.getInstance();
         StackEntity stackEntity;
         if (sm.entityManager.isRegistered(entity))
             stackEntity = sm.entityManager.getStackedEntity(entity);
         else
             stackEntity = sm.entityManager.register(entity);
 
+        if (!EntityHelper.spawnEntity(entity))

```

Comment:
moet dit niet stackEntity zijn? heb de code nog niet in IDE bekijken of gerunt dus could be wrong

Reply:
Nope, we want to spawn the LivingEntity in the world, the Minecraft code base wouldn't understand what to do with a StackEntity.